### ISSUE TITLE
Custom dataset of 2k images - How many epochs are recommended?

### ISSUE THREAD

**Issue by taewookim** [yhenon/keras-frcnn#220]
Setup:

ubuntu 16.04 LTS, 4 vCPU + 30gb ram
nvidia K80 GPU w/12gb memory
CUDNN 8.0 x64
TF version 1.3
Keras version 2.0.9

I'm training on a dataset of 2k images, 5 classes (~400 images per class). How many epochs are recommended ? I noticed after about 18 epochs, the loss was settling below 1, but when i apply the trained weights, I get barely any detection whatsoever.

1) Is there a min. recommended # of images for this architecture?
2) what's the recommended # of epochs for a dataset this size?
3) is there a pretrained weights for COCO dataset?
